### PR TITLE
Remove pexpect

### LINKED ISSUE #195: Why is pexpect included in the source?
Why is pexpect's source code included in the PyLaGriT package?

From a quick glance, it isn't modified (is it?). Why not just list it as a dependency in the `setup.py`?

If a specific version is needed, you can set that in the `setup.py`.

### PR DESCRIPTION
**Pull request by banesullivan**
Resolve #195 

This removes the `pexpect` package from being included - it isn't good practice to have it that way. Version 4.6.0 (the version that was included) is available on PyPI: https://pypi.org/project/pexpect/4.6.0/ and should be installed from `pip`

This also:

- cleans up some line endings
- adds pip files to the gitignore

## Todo

The Python module isn't tested on Travis?... It looks like tests and examples are all written but they aren't verified on Travis.

Should I use this pull request to implement installing the Python module and running those tests and examples to verify everything works?

### CODE REVIEW

File: PyLaGriT/setup.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
 	author_email='dharp@lanl.gov',
 	url='lagrit.lanl.gov',
 	license='LGPL',
+	install_requires=[
+		'pexpect==4.6.0',],
```
Comment (banesullivan) [inline PyLaGriT/setup.py:12]: This is all it takes to use `pexpect`

**Comment by daniellivingston** [comment]
Assuming this has no downstream effects (probably a safe assumption), then I support this PR

**Comment by banesullivan** [comment]
Thanks for the support, @daniellivingston! I'd like to make sure this is all tested before merging... Should I give a go at running all the examples and the test suite on Travis?

**Comment by daniellivingston** [comment]
Travis would likely fail; we have some breaking changes in our test suite at the moment. There is a PyLaGriT test folder that should pass, though.

But work with @dharp on this: PyLaGriT is his project, and he'll be the one to approve/reject/modify this PR :)

**Pull request merged**